Webtest.sh

```diff
@@ -61,14 +69,29 @@ docker exec $DOCKERNAME /usr/local/bin/fortio grpcping localhost
 # Do a grpcping to a scheme-prefixed destination. Fortio should append port number
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping https://fortio.istio.io
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping http://fortio.istio.io
+# Do a grpcping with -cert flag. Fortio should use valid cert.
+docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT fortio.istio.io:443
+docker exec $DOCKERNAME /usr/local/bin/fortio grpcping -cacert $CERT https://fortio.istio.io
 # Do a local grpcping. Fortio should append default grpc port number to destination
 docker exec $DOCKERNAME /usr/local/bin/fortio grpcping localhost
 # pprof should be there, no 404/error
 PPROF_URL="$BASE_URL/debug/pprof/heap?debug=1"
 $CURL $PPROF_URL | grep -i TotalAlloc # should find this in memory profile
+# creating dummy container to hold a volume for test certs due to remote docker bind mount limitation.
+DOCKERVOLID=$(docker create -v $TESTCERTSVOL --name certs istio/fortio.build:v6 /bin/true)
```

Comment:
don't hardcode the build image if possible, also it's v7 now. it should work with the same `istio/fortio:webtest` image no (start another server with no port mapping) ?
I'm not super clear how that volumes-from works exactly ? why do we need 2 images, you can't docker cp into the second one directly - is it just a startup issue (files not being there yet?) ?

Reply:
I added verbosity to the comments in my upcoming commit. Let me know if you still need clarity after reviewing the inline comments.